base/applications/notepad/dialog.c

```diff
@@ -415,6 +415,13 @@ VOID DIALOG_FileNew(VOID)
     }
 }
 
+VOID DIALOG_FileNewWindow(VOID)
+{
+    TCHAR pszNotepadExe[MAX_PATH];
+    GetModuleFileName(NULL, pszNotepadExe, MAX_PATH);
```

Comment:
```suggestion
    GetModuleFileName(NULL, pszNotepadExe, _countof(pszNotepadExe));
```
(or `ARRAYSIZE(pszNotepadExe)` in case `_countof` is not available).

Reply:
done